include/libdnf/module/module_item_container.hpp

```diff
@@ -52,9 +52,27 @@ class ModuleItemContainer {
 
     ModuleItemContainerWeakPtr get_weak_ptr();
 
+    std::vector<std::unique_ptr<ModuleItem>> modules;
+
     // TODO(pkratoch): Maybe make this private later
     void add(const std::string & file_content);
-    std::vector<std::unique_ptr<ModuleItem>> modules;
+
+    // TODO(pkratoch): Implement adding defaults from "/etc/dnf/modules.defaults.d/", which are defined by user.
+    //                 They are added with priority 1000 after everything else is loaded.
+    /// @return Add and resolve defaults.
+    /// @since 5.0
+    //
+    // @replaces libdnf:ModuleItemContainer.hpp:method:ModuleItemContainer.addDefaultsFromDisk()
+    // @replaces libdnf:ModuleItemContainer.hpp:method:ModuleItemContainer.moduleDefaultsResolve()
+    void add_defaults_from_disk();
```

Comment:
Why documenting a void function with a return argument? Is it a placeholder for the future?

Reply:
It's a copy-paste mistake, I removed the `@return`. Thanks for the catch!